gribscan/gribscan.py

```diff
@@ -514,9 +517,13 @@ def consolidate_metadata(refs):
 
 
 def prepend_path(refs, prefix):
-    """Prepend a path-prefix to all target filenames in a given reference filesystem."""
+    """Prepend a path-prefix to all target filenames in a given reference filesystem.
+
+    For absolute paths, the existing parents are overwritten.
```

Comment:
I think this sentence is ambiguous. Does this cover the meaning?
```suggestion
    For absolute target paths, the existing target parents are overwritten.
```